### DISCUSSION TITLE
Save client authenticated state after page reload

### DISCUSSION THREAD

**Discussion by ValeryVerkhoturov** [appwrite/appwrite#5158] [Q&A] [answered]
How to save frontend authenticated state after page reload in Next.js with Web SDK?

**Comment by stnguyen90** [comment, marked as answer]
@ValeryVerkhoturov, I recommend only using client-side code if you don't need server-side rendering (SSR). Then, you can do as described in [this](https://github.com/appwrite/appwrite/discussions/3938#discussioncomment-3746725) discussion. SSR complicates things because you must handle sessions for each user server side.
